lib/supavisor/client_handler.ex

```diff
@@ -382,12 +395,48 @@ defmodule Supavisor.ClientHandler do
         Telem.client_join(:fail, data.id)
         {:stop, {:shutdown, :max_clients_reached}}
 
+      :proxy ->
+        {:ok, %{port: port, host: host}} = Supavisor.get_pool_ranch(data.id)
+
+        auth =
+          Map.merge(data.auth, %{
+            port: port,
+            host: ~c"#{host}",
+            ip_version: :inet,
+            upstream_ssl: false,
+            upstream_tls_ca: nil,
+            upstream_verify: nil
+          })
+
+        {:keep_state, %{data | auth: auth}, {:next_event, :internal, :connect_db}}
+
       error ->
         Logger.error("ClientHandler: Subscribe error: #{inspect(error)}")
         {:keep_state_and_data, {:timeout, 1000, :subscribe}}
     end
   end
 
+  def handle_event(:internal, :connect_db, _, data) do
+    Logger.debug("Try to connect to DB")
```

Comment:
```suggestion
    Logger.debug("ClientHandler: Trying to connect to DB")
```

Nitpick, but it makes more sense as it can be read as "I am trying…".